Add ability to hide side tab bars (#10425)
jupyterlab/jupyterlab: docs/source/user/interface.rst

   :align: center
   :class: jp-screenshot

> The icons and labels column hat allows to switch between tabs is called

Reviewer: ```suggestion
The icons and labels column that allows to switch between tabs is called
```

Author: Thanks @krassowski

> The icons and labels column that allows to switch between tabs is called
> Activity Bar in JupyterLab.

Reviewer 2: ```suggestion
The icons and labels column that enables switching between sidebar activities is called
Activity Bar.
```

Reviewer: It looks like patch from this suggestion got lost during rebase.
